cmd/meroxa/root/apps/remove.go

```diff
@@ -77,7 +81,29 @@ func (r *Remove) Execute(ctx context.Context) error {
 	}
 
 	apps := &Applications{}
-	apps, err := apps.RetrieveApplicationID(ctx, r.client, r.args.idOrName, r.flags.Path)
+	var err error
+
+	config, err := turbine.ReadConfigFile(r.flags.Path)
+	if err != nil {
+		return err
+	}
+
+	r.lang = config.Language
+	if r.turbineCLI == nil {
+		if r.turbineCLI, err = getTurbineCLIFromLanguage(r.logger, r.lang, r.flags.Path); err != nil {
+			if err != nil {
+				return err
+			}
```

Comment:
We can remove the extra error check here:
```suggestion
			return err
```

Reply:
Good point!